PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+Fixes #Issue_Number
+
+>note: The "Fixes #nnn" syntax in the PR description causes
+>GitHub to close the issue when this PR is merged. It saves
+>all of us some work. Not all PRs need to reference an issue.
+>See CONTRIBUTING.md for details.
+
+Summary
+========
+
+Insert short (less than 140 character) summary here.
+
+Please always include this.
```

Comment:
I'm confused on what you mean about insert short summary here and long summary here.